aiida_quantumespresso/parsers/pp.py

```diff
@@ -217,7 +217,7 @@ def detect_important_message(logs, line):
                 split_line = line.split('=')
                 if 'negative/imaginary' in line:  # QE6.1
                     output_dict['negative_core_charge'] = float(split_line[-1].split()[0])
-                    output_dict['imaginary_core_charge'] = float(split_line.split()[-1])
+                    output_dict['imaginary_core_charge'] = float(split_line[-1].split()[-1])
```

Comment:
This is apparently not being tested, correct? I guess that is because the output files we have are not from QE 6.1. However, why is there no `imaginary_core_charge` for the QE 6.4 branch?

Reply:
> This is apparently not being tested, correct?

Looks like it wasn't.

> However, why is there no imaginary_core_charge for the QE 6.4 branch?

I don't know why, I can just say that there are no imaginary charges for QE 6.4, indeed:
https://github.com/aiidateam/aiida-quantumespresso/blob/a52266d096afe48dbc6b38b63ac17a9989dd12fe/tests/parsers/fixtures/pp/default_3d/aiida.out#L32

I should also mention, that in `QE 6.5` this line is not present at all, while for  `6.3` it is present, and looks as follows:
```
Check: negative/imaginary core charge=   -0.000001    0.000000
```